autopts/bot/iut_config/zephyr.py

```diff
@@ -153,7 +153,7 @@
         "test_cases": [
             'VOCS', 'VCS', 'AICS', 'IAS', 'PACS', 'ASCS', 'BAP', 'HAS', 'CSIS', 'MICP',
             'MICS', 'VCP', 'MCP', 'CAP', 'BASS', 'GMCS', 'CCP', 'HAP', 'TBS', 'GTBS',
-            'TMAP',
+            'TMAP', 'PBP', 'GMAP', 'CAS',
```

Comment:
While you are at it, please also add `CSIP` and `MCS` (even if we are not running that yet),